pkg/mqtrigger/scalermanager.go

```diff
@@ -0,0 +1,548 @@
+package mqtrigger
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	fv1 "github.com/fission/fission/pkg/apis/core/v1"
```

Comment:
We maintain three set of imports - Core Go, External+K8S and lastly fission, please take a look at for example https://github.com/fission/fission/blob/master/pkg/builder/builder.go

Reply:
Done